src/xlat-i/xi-Parameter.C

```diff
@@ -508,19 +629,21 @@ void ParamList::beginUnmarshall(XStr& str) {
     str << "*/\n";
     str << "  PUP::fromMem implP(impl_buf);\n";
     if (hasRdma()) {
-      str << "#if CMK_ONESIDED_IMPL\n";
-      str << "  char *impl_buf_begin = impl_buf;\n";
-      str << "  int impl_num_rdma_fields; implP|impl_num_rdma_fields; \n";
-      str << "  int impl_num_root_node; implP|impl_num_root_node;\n";
-      callEach(&Parameter::beginUnmarshallRdma, str, true);
-      str << "#else\n";
-      callEach(&Parameter::beginUnmarshallRdma, str, false);
-      str << "#endif\n";
-      if (hasRecvRdma()) {
-        str << "  CkNcpyBufferPost ncpyPost[" << entry->numRdmaRecvParams << "];\n";
-        for(int index=0; index < entry->numRdmaRecvParams; index++) {
-          str << "  ncpyPost[" <<index<<  "].regMode = CK_BUFFER_REG;\n";
-          str << "  ncpyPost[" <<index<<  "].deregMode = CK_BUFFER_DEREG;\n";
```

Comment:
Where has this code to set default registration/de-registration mode been moved to?

Reply:
I had fixed the constructor of `CkNcpyBufferPost` to set them, but these lines should be brought back because now there is a separate `CkDeviceBufferPost`. Will make the update.